src/test/java/uk/ac/ebi/eva/pipeline/jobs/steps/GenerateVepAnnotationStepTest.java

```diff
@@ -92,10 +103,10 @@ public void shouldGenerateVepAnnotations() throws Exception {
         assertEquals(BatchStatus.COMPLETED, jobExecution.getStatus());
 
         // And VEP output should exist and annotations should be in the file
-        File vepOutputFile = new File(URLHelper.resolveVepOutput(vepOutputFolder.getAbsolutePath(), STUDY_ID, FILE_ID));
-
-        assertTrue(vepOutputFile.exists());
-        assertEquals(537, JobTestUtils.getLines(new GZIPInputStream(new FileInputStream(vepOutputFile))));
+        assertTrue(vepOutput.exists());
+        int EXTRA_ANNOTATIONS = 1;
+        assertEquals(300 + EXTRA_ANNOTATIONS,
+                JobTestUtils.getLines(new GZIPInputStream(new FileInputStream(vepOutput))));
```

Comment:
the VepAnnotationFileWriterTest and VepProcessTest could not work with the existing mockvep.pl, so I added two more versions of that script. After that, for the other integration tests I had to modify the old mockvep, and now this test can work with both scripts. 

The old script now takes all data from standard input, ignores it, and writes a hardcoded (correct) annotation. the new script copies the input into the output in batches, adding "annotated" to each line (not actual annotation), and adds 1 extra line (vep can do this, and we need to check that we support that behaviour). There's a third script similar to the second one that has sleeps for the timeout checks